### PR TITLE
add liquibase changeset for adding new crawler sources via pull request

### PR DESCRIPTION
**Pull request by kkehl-usgs**

### CODE REVIEW

File: liquibase/changeLogs/nldi/nldi_data/changeLog.xml [hunk 1]
```diff
@@ -14,4 +14,23 @@
 	<include file="functions/changeLog.xml" relativeToChangelogFile="true"/>
 	<include file="grants.sql" relativeToChangelogFile="true"/>
 
+
```
Comment (dsteinich) [inline liquibase/changeLogs/nldi/nldi_data/changeLog.xml:17]: I'd prefer to see this in it's own director/file - as an example have a peek at https://github.com/usgs/aqts-capture-db/tree/master/liquibase/changeLogs/capture/data

We have been moving to yml changesets. I think a mix works, and if so, the new ones should be yml.

We can also use the built in liquibase delete - https://github.com/usgs/aqts-capture-db/blob/master/liquibase/changeLogs/capture/data/aqCompIdToStatCd/changeLog.yml

File: liquibase/changeLogs/nldi/nldi_data/crawler_source.csv [hunk 2]
```diff
@@ -0,0 +1,2 @@
+'crawler_source_id','source_name','source_suffix','source_uri','feature_id','feature_name','feature_uri','feature_reach','feature_measure','ingest_type'
+6,'Water Data Exchange 2.0 Sites','wade','https://www.hydroshare.org/resource/5f665b7b82d74476930712f7e423a0d2/data/contents/wade.geojson','feature_id','feature_name','feature_uri',NULL,NULL,'point'
```
Comment (dsteinich) [inline liquibase/changeLogs/nldi/nldi_data/crawler_source.csv:2]: This file also needs to include the sources that are currently in prod.

Comment (kkehl-usgs) [reply liquibase/changeLogs/nldi/nldi_data/crawler_source.csv:2]: This was the list you provided me.  I didn't have access to the prod database.  Is it not complete?

File: liquibase/changeLogs/nldi/nldi_data/changeLog.xml (hunk 1 above)
Comment (kkehl-usgs) [reply liquibase/changeLogs/nldi/nldi_data/changeLog.xml]: ok done

**Pull request merged**